### PR TITLE
Add fsType to aws-ebs

### LINKED ISSUE #10038: Issue with k8s.io//docs/concepts/storage/storage-classes/ [File: /content/en/docs/concepts/storage/storage-classes.md]
**This is a...** 

- [ ] Feature Request
- [X] Bug Report

**Problem:**
The option for setting the filesystem is only explained for ScaleIO and Portworx Volume, while AWS EBS supports it as well, according to my tests and the code (https://github.com/kubernetes/kubernetes/blob/e40ffd71972b820aeed283946e880aa2479f8524/pkg/volume/aws_ebs/aws_ebs.go#L117)

**Proposed Solution:**
Please add the `fsType` option to the documentation for AWS EBS as well. Took me a long time to figure it out, would've been nice if it was documented!

**Page to Update:**
https://kubernetes.io//docs/concepts/storage/storage-classes/

### PR DESCRIPTION
**Pull request by makocchi-git**
- closes #10038
- revise indent for correct rendering

### CODE REVIEW

**Comment by makocchi-git** [comment]
[preview is here](https://deploy-preview-10041--kubernetes-io-master-staging.netlify.com/docs/concepts/storage/storage-classes/)
[original](https://kubernetes.io//docs/concepts/storage/storage-classes/)

**Comment by Bradamant3** [comment]
/approve for doc review

tech review, anyone?

File: content/en/docs/concepts/storage/storage-classes.md [hunk 1]
```diff
@@ -11,14 +11,12 @@ weight: 30
...
 {{% capture overview %}}
 
-This document describes the concept of `StorageClass` in Kubernetes. Familiarity
+This document describes the concept of a StorageClass in Kubernetes. Familiarity
 with [volumes](/docs/concepts/storage/volumes/) and
 [persistent volumes](/docs/concepts/storage/persistent-volumes) is suggested.
 
 {{% /capture %}}
 
-{{< toc >}}
```
Comment (msau42) [inline content/en/docs/concepts/storage/storage-classes.md]: Can we keep the toc?

Comment (Bradamant3) [reply content/en/docs/concepts/storage/storage-classes.md]: It's added automatically, without explicitly specifying it.

Comment (Bradamant3) [reply content/en/docs/concepts/storage/storage-classes.md]: You can see it here if you like: https://deploy-preview-10041--kubernetes-io-master-staging.netlify.com/docs/concepts/storage/storage-classes/

Comment (msau42) [reply content/en/docs/concepts/storage/storage-classes.md]: Ah ok, I was confusing it with the Volumes page, which has a TOC-like list of all the volume types.

**Pull request merged**